spec/models/video_spec.rb

```diff
@@ -1,43 +1,35 @@
 require 'spec_helper'
 
 describe Video do
+  it { should belong_to(:category) }
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+
+  let(:first_video) { Fabricate(:video, title: "The first title", description: "The first description", created_at: 1.day.ago) }
+  let(:second_video) { Fabricate(:video, title: "The second title", description: "The second description") }
+
   it "saves itself" do
-    first_video = Video.create(title: "The first title", description: "The first description")
     first_video.save
     expect(Video.first).to eq(first_video)
   end
 
-  it { should belong_to(:category) }
-  it { should validate_presence_of(:title) }
-  it { should validate_presence_of(:description) }
-
   it "returns search by title by title" do
```

Comment:
The problem of this refactoring is with `let` these items are not created now, as before you had them in the database. When you think about it - this test basically proves nothing. You should have at least 1 record in the database and search for something like "hello" to prove the faulty search with return an empty array.